.github/workflows/test_dht_node.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: actions/checkout@v3
       
       - name: Lint
-        run: cd dht-node && yarn && yarn run lint
+        run: cd database && yarn && cd ../backend && yarn && yarn run lint
```

Comment:
```suggestion
        run: cd dht-node && yarn && yarn run lint && cd ../database && yarn && yarn run lint && cd ../backend && yarn && yarn run lint
```

Reply:
This line makes sure the database has all required `node_modules` and built files so testing/linting works properly.

Only the current project (e.g. `dht_node`) is linted.

Reply:
To you are right - there was a mistake in this like - it should be `cd ../dht-node`